File: Semester 2/UPM/Pemrograman/Soal Pagi 2022/Nomor1.java
import java.util.Scanner;

public class Nomor1 {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        
        // Input Angka
        System.out.print("Input Sebuah Angka : ");
        int jumlah = in.nextInt();
        int i, sisa, bagi;
        i = 1;

        // Proses Loop Menentukan Mod & Hasil Bagi
        while (i <= 10) {
          sisa = jumlah % i;
          bagi = (int) jumlah / i;
          System.out.printf("%s : %s = %s sisa %s\n", jumlah, i, bagi, sisa);
          i++;
        }
   }
}

File: Semester 1/UAS/Pemrograman/Soal Pagi 2022/Nomor1.pas
program Nomor1;
uses crt;

var a, b, sisa, hasil: integer;

begin
    clrscr;
    write('Input Sebuah Angka : '); readln(b);
    writeln('=======================');
    a := 1;
    while (a <= 10) do
    begin
        hasil := b div a;
        sisa := b mod a;
        writeln(b, ' : ', a, ' = ', hasil, ' sisa ', sisa);
        a := a + 1;
    end;
    readln;
end.
